src/commands/txCommands/util/index.js

```diff
@@ -209,6 +210,36 @@ const uploadTrnsMaster$ = ([lang_tag, content]) =>
 		resourceContent(MASTER_RESOURCE, content)
 	);
 
+const uploadTranslation$ = ([lang_tag, content]) =>
+	Rx.Observable.bindNodeCallback(tx.translationStringsPutMethod.bind(tx))(
+		PROJECT_MASTER,
+		MASTER_RESOURCE,
+		lang_tag,
+		content
+	);
+
+const uploadTranslationsForKeys$ = keys => allLocalPoTrns$
+	.flatMap(([lang_tag, content]) => filterPoContentByKeys$(keys,content)
+		.map(poToUploadFormat)
+		.map(filteredTrnObj => [lang_tag,filteredTrnObj])
+	)
+	.flatMap(uploadTranslation$)
+
+const poToUploadFormat = trnObj =>
+	Object.keys(trnObj)
+		.reduce((arr,key) => 
+			arr.concat({'key':key,'translation':trnObj[key].msgstr[0]}),[]) 
+
+const filterPoContentByKeys$ = (keys, poContent) => Rx.Observable.of(poContent)
+	.map(trnObj => _.pick(trnObj, keys)) // return object with specified keys only
+	.filter(obj => Object.keys(obj).length) // remove empty objects
+
+const poToReactIntlFormat = trns => Object.keys(trns)
+	.reduce((messages, trnKey) => {
```

Comment:
trivial, but you can use [`_.mapValues`](https://lodash.com/docs/4.17.4#mapValues) on an object to make a copy of the object with the `value` of each key transformed